nixos/modules/services/databases/pgmanage.nix

```diff
@@ -133,10 +132,10 @@ in {
 
     dataRoot = mkOption {
       type = types.str;
-      default = "/var/lib/postage";
+      default = "/var/lib/pgmanage";
```

Comment:
same here, this is probably more important to keep the old behaviour by default.

Reply:
We have `stateVersion` to cover this.